distribution/ecs/quickwit/quickwit-indexer.tf

```diff
@@ -20,6 +20,9 @@ module "quickwit_indexer" {
   enable_cloudwatch_logging      = var.enable_cloudwatch_logging
   service_config                 = var.quickwit_indexer
   quickwit_index_s3_prefix       = local.quickwit_index_s3_prefix
+  # Longer termination grace period for indexers because their ingest services
```

Comment:
Technically, we're not waiting for the data in-flight to be committed to the WALs. We're waiting for the shards to be drained, i.e., indexed by local or remote indexing pipelines.